cpp/benchmarks/scorep_measuring.md

```diff
@@ -0,0 +1,40 @@
+# Performance measuring with Score-P
+
+This file explains how to use Score-P with GCC. For other compilers this might look different or not work correctly.
+Once you have Score-P you can build the abm with: 
+```
+export SCOREP_WRAPPER=off #disable scorep during cmake configure
+cmake .. -DMEMILIO_ENABLE_OPENMP=ON -DCMAKE_CXX_COMPILER=${Path to scorep}/scorep-mpic++ -DCMAKE_C_COMPILER=${Path to scorep}/scorep-mpicc -DCMAKE_BUILD_TYPE=RelWithDebInfo
+export SCOREP_WRAPPER=on #reenable scorep
+export SCOREP_WRAPPER_INSTRUMENTER_FLAGS="--pomp --thread=omp --mpp=none --instrument-filter=${Path to filter}/filter-main"
+cmake --build . -j --target abm_simulation
+```
+
+The [filter file](./filter-main) is designed to include only the interesting information about the abm. It filters (i.e. measures) only the basic framework of the abm, that is only the simulation part up to the loop over time. The measurement of smaller functions increases the overhead of the profiling and are therefore neglected in the current filter. This profile may give a hint on which part of the abm you should investigate further. To measure additional functions you add ```INCLUDE *Class::myfunction*``` to the [filter file](./filter-main). 
```

Comment:
I think your explanation sounds like overhead is only a problem because the profiling will take longer. But the problem is that profiles with large runtime overhead are unreliable, since the time measured by the profiler is not the time that the functions would take in normal operation.

Reply:
added the sentence.